include/boost/numeric/ublas/matrix_sparse.hpp

```diff
@@ -368,7 +368,7 @@ namespace boost { namespace numeric { namespace ublas {
 
         // Reserving
         BOOST_UBLAS_INLINE
-        void reserve (size_type non_zeros, bool preserve = true) {
+				void reserve (size_type non_zeros, bool /*preserve = true*/) {
```

Comment:
please don't change the indentation. (this might be from your editor's config. Please don't do that. :-) )

Reply:
Oh, and there is also a bug: your change removes the default value, so existing code calling `reserve(N)` will presumably no longer work. You should comment out only the argument name, e.g.
"void reserve(size_type non_zeros, bool /*preserve*/ = true)".

Reply:
`preserve` is not used within the function body.

Reply:
Perhaps, but do you know all the code that calls this function ? :-)

Reply:
no, but it does not change if the calling function sets preserve, it does not have an effect. Actually it should be commented out to indicate that it is not used.

Reply:
I understand. My point is: if a user calls the function with two arguments, it now generates a compilation error, because after your change the function only takes one argument.